src/tests/performVerifySorting.spec.js

```diff
@@ -0,0 +1,38 @@
+const { expect } = require('@playwright/test');
+const { test } = require('../fixture');
+const {login, password}  = require('../auth/user.json');
+
+test.describe('Inventory Sorting', () => {
+    test.beforeEach(async ({ loginPage }) => {
+        await loginPage.navigate();
+        await loginPage.performLogin(login, password);
+    });
+
+    test('Sort items by Name (A to Z)', async ({ shopingCartPage }) => {
```

Comment:
really great job!
now let's use https://playwright.dev/docs/test-parameterize to make it shorter and easier. You will have only one `test` function instead of 4